sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py

```diff
@@ -884,149 +1212,6 @@ def from_string(cls, permission):
-        Enable/disable 'Temporary' attribute for FILE
-    :ivar bool offline:
-        Enable/disable 'Offline' attribute for DIRECTORY or FILE
-    :ivar bool not_content_indexed:
-        Enable/disable 'NotContentIndexed' attribute for DIRECTORY or FILE
-    :ivar bool no_scrub_data:
-        Enable/disable 'NoScrubData' attribute for DIRECTORY or FILE
-    """
-    def __init__(self, read_only=False, hidden=False, system=False, none=False, directory=False, archive=False,
-                 temporary=False, offline=False, not_content_indexed=False, no_scrub_data=False):
-
-        self.read_only = read_only
-        self.hidden = hidden
-        self.system = system
-        self.none = none
-        self.directory = directory
-        self.archive = archive
-        self.temporary = temporary
-        self.offline = offline
-        self.not_content_indexed = not_content_indexed
-        self.no_scrub_data = no_scrub_data
-        self._str = (('ReadOnly|' if self.read_only else '') +
-                               ('Hidden|' if self.hidden else '') +
-                               ('System|' if self.system else '') +
-                               ('None|' if self.none else '') +
-                               ('Directory|' if self.directory else '') +
-                               ('Archive|' if self.archive else '') +
-                               ('Temporary|' if self.temporary else '') +
-                               ('Offline|' if self.offline else '') +
-                               ('NotContentIndexed|' if self.not_content_indexed else '') +
-                               ('NoScrubData|' if self.no_scrub_data else ''))
-
-    def __str__(self):
-        concatenated_params = self._str
-        return concatenated_params.strip('|')
-
-    @classmethod
-    def from_string(cls, string):
-        """Create a NTFSAttributes from a string.
-
-        To specify permissions you can pass in a string with the
-        desired permissions, e.g. "ReadOnly|Hidden|System"
-
-        :param str string: The string which dictates the permissions.
-        :return: A NTFSAttributes object
-        :rtype: ~azure.storage.fileshare.NTFSAttributes
-        """
-        read_only = "ReadOnly" in string
-        hidden = "Hidden" in string
-        system = "System" in string
-        none = "None" in string
-        directory = "Directory" in string
-        archive = "Archive" in string
-        temporary = "Temporary" in string
-        offline = "Offline" in string
-        not_content_indexed = "NotContentIndexed" in string
-        no_scrub_data = "NoScrubData" in string
-
-        parsed = cls(read_only, hidden, system, none, directory, archive, temporary, offline, not_content_indexed,
-                     no_scrub_data)
-        parsed._str = string  # pylint: disable = protected-access
-        return parsed
-
-
 def service_properties_deserialize(generated: GeneratedStorageServiceProperties) -> Dict[str, Any]:
```

Comment:
One overall comment about this file since things got moved around a bunch. Can you make sure the `...Paged` models are always right underneath their equivalent non-paged models? So `FilePropertiesPaged` is right under `FileProperties` for example. Just to keep them together. They may already be but just wanted to be sure.

Reply:
Doubled checked all files, they're all `X` followed by `XPaged`. The only exception is in `aio` where obviously `XPaged` is detached from `X`. Let me know if you see anything different or concerning.